platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/ChallengeTemplate.java

```diff
@@ -13,13 +16,18 @@
  */
 @Entity
 @Table(name = "challenge_template")
-public class ChallengeTemplate extends Assignment {
+public class ChallengeTemplate extends Assignment implements CanonicalEntity{
     @ManyToOne
     private Endpoint endpoint;
 
     @ManyToOne
     private Organization organization;
 
+    @NotNull
+    @NotEmpty
+    @Column(name = "canonical_name", unique = true, nullable = false)
+    private String canonicalName;
```

Comment:
dont see any migration scripts for that, and is that really needed? why only old CCC games need names?